webapp/src/modules/mapBuilder.js

```diff
@@ -280,6 +280,8 @@ export default class MapBuilder {
   }
 
   newMarker(id, bike, vehicleType, providers, precisionFactor) {
+    // use offset coordinates when available
+    bike.c = !bike.o ? bike.c : bike.o;
```

Comment:
Can't modify **bike.c** since it is API uses it to GET vehicle information from back-end. Will need to get coordinates variables from either bike.c or bike.o conditionally.

Reply:
> Update the backend to use offsets in o rather than absolute coordinates

@rgalanakis Does this mean refactoring the response as lat, lng?